src/main/java/jenkins/plugins/git/traits/CleanBeforeCheckoutTrait.java

```diff
@@ -50,6 +50,8 @@ public CleanBeforeCheckoutTrait() {
 
     /**
      * Stapler constructor.
+     *
+     * @param extension the option to clean subdirectories which contain git repositories.
```

Comment:
@darxriggs I'm open to a better way to describe this parameter and the same parameter on the other trait.

Reply:
If you have a look at the other traits they more or less only link to the type of the parameter. Maybe keep it like this and not duplicate the description of the extension or come up with another description for it.